src/libmeasurement_kit/common/utils.cpp

```diff
@@ -19,6 +19,20 @@ struct timeval;
 
 namespace mk {
 
+#ifdef _WIN32
+tm *gmtime_r(const time_t *timep, tm *result) {
```

Comment:
TODO(@bassosimone): the previous attempt at porting to Windows had a more robust impl.